File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        int longest = 0;
        unordered_set<int> seen;

        for (int i = 0; i < nums.size(); i++) {
            seen.insert(nums[i]);
        }

        for (int i = 0; i < nums.size(); i++) {
            int length = 1;
            while (seen.find(nums[i] + length) != seen.end()) {
                length++;
            }
            longest = max(longest, length);
        }

        return longest;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        seen = {}
        for i in range(len(nums)):
            t = target - nums[i]
            if t in seen:
                return [seen[t], i]
            seen[nums[i]] = i